src/linux/minidump_writer.rs

```diff
@@ -189,7 +189,7 @@ impl MinidumpWriter {
     ) -> Result<()> {
         // A minidump file contains a number of tagged streams. This is the number
         // of streams which we write.
-        let num_writers = 15u32;
+        let num_writers = 17u32;
 
```

Comment:
This should be 16 I believe.

Reply:
While the current code only collects `limits`, I dont remember if we need to augment things to match the work @gabrielesvelto did to help us collect also the FDs in use (I've asked him on matrix, but at the same time I'm dealing with one of my display dying its DisplayPort).

Reply:
Sure, but that branch is not merged, it will be 17 after both are merged since they each add a single stream, but this would be a bug if this lands before that PR, and you'd be surprised at how many people follow HEAD that you don't find out about until you break them.

Reply:
Right, I just saw #94 which answers to my question.